mpas_analysis/ocean/nino34_index.py

```diff
@@ -203,18 +247,28 @@ def compute_nino34_spectra(nino34Index):  # {{{
 
     Last Modified
     -------------
-    03/23/2017
+    04/09/2017
     """
 
     # Move nino34Index to numpy to allow functionality with scipy routines
     ninoIndex = nino34Index.values
     window = signal.tukey(len(ninoIndex), alpha=0.1)
     f, Pxx = signal.periodogram(window * ninoIndex,
-                                1.0 / constants.sec_per_month,
-                                scaling='spectrum')
-
-    # computes power spectra, smoothed with 5 point running mean
-    pxxSmooth = _running_mean(pd.Series(Pxx))
+                                1.0 / constants.sec_per_month)
+
+    # computes power spectra, smoothed with a weighted running mean
+    nwts = max(1, int(7*len(ninoIndex) / 1200))
+    # verify window length is odd, if not, add 1
+    if nwts % 2 == 0:
+        nwts += 1
+    # Calculate the weights for the running mean
+    # Weights are from the Daniell Window
```

Comment:
Is this the Daniell Window?  I recall you had said it wasn't quite the Daniell Window when we talked about this a while back and just want to double-check the comment is ok.